chain/syncer_test.go

```diff
@@ -193,33 +195,33 @@ func requireSetTestChain(t *testing.T, con consensus.Protocol, mockStateRoots bo
 func loadSyncerFromRepo(t *testing.T, r repo.Repo, dstP *SyncerTestParams) (*chain.Syncer, *th.TestFetcher) {
 	powerTable := &th.TestView{}
 	bs := bstore.NewBlockstore(r.Datastore())
-	cst := hamt.NewCborStore()
+	cst := &hamt.CborIpldStore{Blocks: bserv.New(bs, offline.Exchange(bs))}
```

Comment:
These changes aren't strictly necessary but put our tests in a state where the global IPLD store approaches global.  I can change back if desired.